src/configs/data/mvtec_loco.yaml

```diff
@@ -0,0 +1,18 @@
+class_path: anomalib.data.MVTecLoco
+init_args:
+  root: ./datasets/MVTec_LOCO
+  category: breakfast_box
+  image_size: [256, 256]
+  center_crop: null
+  normalization: imagenet
+  train_batch_size: 32
+  eval_batch_size: 32
+  num_workers: 8
+  task: SEGMENTATION
+  transform_config_train: null
+  transform_config_eval: null
+  test_split_mode: FROM_DIR
+  test_split_ratio: 0.2
+  val_split_mode: FROM_DIR
+  val_split_ratio: 0.5
```

Comment:
I believe using 'null' might lead to ambiguity with the 'none' option, no? As far as I know, 'none' is supposed to be used to skip the validation/testing stage. You can refer to this issue context:https://github.com/openvinotoolkit/anomalib/issues/1629.

I thought 'FROM_DIR' would be suitable since in the case of 'test_split_mode', as also used by other datasets like MVTec, etc., the FROM_DIR option retrieves the test sample from a fixed test set. Even if FROM_DIR is used, the split ratio actually ignored because the test set already provides both normal and abnormal samples.

But yeah, the `*_split_ratio` is quite confusing because it is actually ignored. So it might be better to remove the split ratio from the `yaml` file.

https://github.com/openvinotoolkit/anomalib/blob/2cdeb82eb79348a23325fb3d873862775366faa2/src/configs/data/mvtec.yaml#L14-L17

Reply:
`test_split_mode: FROM_DIR` and `val_split_mode: FROM_DIR` is fine, i was just referring to this: 

>  if FROM_DIR is used, the split ratio actually ignored because the test set already provides both normal and abnormal samples.

>  So it might be better to remove the split ratio from the yaml file.

@ashwinvaidya17 is that the convention?

Reply:
Ah i see, I misunderstood your comments. Yeah, i wonder why the test_split_ratio and val_split_ratio are provided in the config file of MVTec and some other datasets even though it seems that those are ignored.

Reply:
> We could consider to set them default as null and raise a warning when a value is provided but is ignored.

@djdameln Got it. Should we just set them as null in this PR? I mean, this part `raise a warning when a value is provided but is ignored.`, I think out of the scope of this PR since it is actually the problem of all existing datasets, right? Not only for this specific dataset.